_includes/mermaid.html

```diff
@@ -0,0 +1,10 @@
+<script src="https://unpkg.com/mermaid@8.9.3/dist/mermaid.min.js"></script>
```

Comment:
@ocefpaf does the version need to change here too?

Reply:
Actually, only here. Above it should be jinja if-clause in https://github.com/ioos/documentation-theme-jekyll/pull/16#discussion_r1457795180

Reply:
Do you mind proposing those changes? I'm not following.

Reply:
Sorry, Will do.